.ci/pytorch/test.sh

```diff
@@ -991,8 +991,15 @@ test_docs_test() {
 
 test_executorch() {
   # Test torchgen generated code for Executorch.
-  echo "Testing Executorch op registration"
+  echo "Testing ExecuTorch op registration"
```

Comment:
Per the chat with @larryliu0820, I will create an issue to resurrect `test_edge_op_registration` added by https://github.com/pytorch/pytorch/pull/89596 in the new ET world

Reply:
https://github.com/pytorch/pytorch/issues/113507